api/openapi-spec/swagger.json

```diff
@@ -78637,6 +78637,10 @@
      "webhook"
     ],
     "properties": {
+     "eventVersion": {
+      "description": "EventVersion is the version of the event to emit. Default v1",
```

Comment:
Just a query that right now it is handling only v1 vesion, So what happend if user specified v1alpha1 or else. Will it respond to the same? Do we need to handle for other versions? Thanks in advance